### DISCUSSION TITLE
Ping-Pong Functionality

### DISCUSSION THREAD

**Discussion by adityakrk** [quarkusio/quarkus#41226] [Community]
Hi ,
I have implemented ping-pong functionality for jakarta.websocket using quarkus . so i wrote some TestClient to check the functionality .
Firstly i have connected to the server using TestClient once the connection is established onOpen method initiate the heartbeat to sendPing to the client for every 30sec. But client is not receiving the PongMessage instead of that server receiving the PongMessage .
 ```
@OnMessage
    public void onPongMessage(Session session, PongMessage pongMessage) {
}
```
But i want client should receive the pongmessage if server sends ping .It is not happening in my case . Please find the details of the version i have used and suggest me the version which work as excepted .
Quarkus Version : 3.4.1
io.quarkus:quarkus-websockets :3.4.1

**Comment by codespearhead** [comment]
@adityakrk Duplicate of #41228 .

Can you [close it](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#closing-a-discussion)?
